### DISCUSSION TITLE
Quarto does not render after Julia upgrade

### DISCUSSION THREAD

**Discussion by bjkwak** [quarto-dev/quarto-cli#9534] [Q&A] [answered]
### Description

I have a Quarto document that contains Julia code chunks.

Today, I upgraded my Julia kernel from 1.10.1 to 1.10.2.
After the upgrade, I can no longer render my Quarto document.
I am on Windows 10.

```
> quarto --version
1.4.551

>quarto render my_quarto_doc.qmd --to html

Starting julia-1.10 kernel...ERROR:

[WinError 2] The system cannot find the file specified
```

The document has not been changed.  The only difference is the new Julia kernel.

In Julia REPL, I entered package mode and checked the status of the packages, and the packages are up to date.

```julia
(@v1.10) pkg> status
tatus `C:\Users\BJK\.julia\environments\v1.10\Project.toml`
 [7073ff75] IJulia v1.24.2
 [c3e4b0f8] Pluto v0.19.41
 
@v1.10) pkg>
``` 

Please help me.
Thanks.

**Comment by mcanouil** [comment, marked as answer]
I'm afraid, that's unrelated to Quarto but related to your kernel setup which does not change when upgrading Julia.

You need to fix your Jupyter setup:
```sh
jupyter kernelspec list
```

Go to the path listed corresponding to Julia and check what is inside `kernel.json`.

See also <https://quarto.org/docs/computations/julia.html#installation>.

**Reply by bjkwak** [reply]
Thank you so much!  I was able to solve the problem following your instructions.

In case someone else encounter similar problem, here are the steps I took:

1. Run `jupyter kernelspec list`.  I gave me the location of `kernel.json`.

2. Checked the content of `kernel.json`.  It had old Julia kernel info.

3. To let Jupyter know that Julia kernel has been upgraded, I opened Julia REPL, and run the following commands:

```julia
using IJulia
installkernel("Julia")
```

4. Checked the file `kernel.json` to make sure it has updated info.

Thank you again for your prompt help.
BJ

**Comment by cscheid** [comment]
@mcanouil @cwickham We really need to add this to our FAQ or something in our docs. I know we have a policy of not polluting our docs to describe fixes for other systems, but it's such a common fail case for jupyter users that I think we'll have to educate them ourselves.

**Reply by mcanouil** [reply]
Agreed.

(There is also the "use your browser developer/inspect mode to customise your document" thing also)

**Reply by cderv** [reply]
We discussed adding  a FAQ part in our documentation, so maybe this is good first candidate. It would allow to share some similar information. Note sure where we want to put it exactly in the website.
